code/modules/mapping/dmm_suite/dmm_preloader.dm

```diff
@@ -65,22 +56,26 @@ GLOBAL_REAL_VAR(dmm_preloader_target)
 
 	// handle post processing, so things like directions on subtypes don't break.
 	// only do everything if necessary.
-	if(preloader_local.turn_angle != 0 && what.preloading_dir(preloader_local))
+	var/datum/dmm_context/loading_context_local = preloader_local.loading_context
+	var/turn_angle = loading_context_local.loaded_orientation_turn_angle
+	if(turn_angle != 0 && what.preloading_dir(preloader_local))
```

Comment:
```suggestion
	if(turn_angle && what.preloading_dir(preloader_local))
```
Should just be able to check for truthyness here

Reply:
Yup, you're right!